pkgs/applications/networking/browsers/chromium/browser.nix

```diff
@@ -80,6 +80,6 @@ mkChromiumDerivation (base: rec {
     platforms = platforms.linux;
     hydraPlatforms = if channel == "stable" then ["aarch64-linux" "x86_64-linux"] else [];
     timeout = 172800; # 48 hours
-    broken = channel == "dev";
+    broken = channel != "stable";
```

Comment:
Can you add a comment similar to the commit message here?

Reply:
Done ;)
But may I ask if we have a good reason for this? I mean it's easier to find, but I'd probably still prefer using `git-blame` to keep the diff more minimal and mainly to avoid outdated comments (which unfortunately happens sometimes).

Reply:
yeah, I just thought in that case it might be helpful, if people stumble over it and don't immediately check `git blame`.

Reply:
Ok, I see. And thinking more about it I agree that it's probably better in this case as it won't change much anyway :)